addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -853,6 +853,82 @@ QUnit.test('Form view not scrolled when switching record', async function (asser
     );
 });
 
+QUnit.test('chatter remove attachment', async function (assert) {
```

Comment:
Would name test like an assertion comment, so that it's quite obvious what this test is meant for.
For example:
_"Attachments that have been unlinked from server should be visually unlinked from record"_
Note that this name is not super precise, because we could assume that initial load was wrong. So could be appropriate to give more context as comment in heading of the test. E.g.

```js
QUnit.test('Attachments that have been unlinked from server should be visually unlinked from record', async function (assert) {
   // Attachments that have been fetched from a record at certain time and then
   // removed from the server should be reflected on the UI when the current
   // partner re-access this record.
```